### DISCUSSION TITLE
DIA ion quantification - diaPASEF

### DISCUSSION THREAD

**Discussion by rodvrees** [Proteobench/ProteoBench#438] [Potential new module to discuss]
# Aim of the new module

## What is its aim?
The same as the currently in development DIA ion quant module

## What is the added benefit?
- We have discussed that orbitrap AIF data, currently used in the ion quant module, is probably not considered as the state-of-the art in DIA acquisition, and it's relevance is arguably fading compared to diaPASEF- and Astral-acquired data. End-users, tool developers (and ProteoBench manuscript reviewers ;-) ), might prefer to see more state-of-the art data analysis being benchmarked. 

- It is possible that search engines perform very well on data acquired from one type of acquisition and poorer on other types. So making conclusions on the performance of a certain search engine on one type of  acquisition is not always safe. Furthermore, differences in settings may impact performance of a tool in one acquisition mode more, or less, than in another. Being able to benchmark data analysis workflows on different DIA acquisition methods prevents such biases

## Why there could be reservations about adding this module.
I understand that having a module that just replicates another one apart from the data kind of sets a precedent for having a lot of duplicated modules for all kinds of different data acquisition methods, for all kinds of benchmarking analyses. I think we can also maybe discuss if instead of having a fully new module for this, we can think of a way to combine different data acquisition modes in the same module. For example, having a selection box within a module, with which the user can select the specific data acquisition, and dependent on this selection the relevant plots are shown, and the new datapoints are added to the relevant plot.

# full description of the new module

See current DIA ion quant module.
Implementation of this module should be very straight-forward, as the code for the currently existing module can be re-used.

The data that can be used is also the same as the current module, the only difference is that it was acquired on the timsTOF in diaPASEF mode:
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_A_Sample_Alpha_01.d.zip
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_A_Sample_Alpha_02.d.zip
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_A_Sample_Alpha_03.d.zip
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_B_Sample_Alpha_01.d.zip
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_B_Sample_Alpha_02.d.zip
https://ftp.pride.ebi.ac.uk/pride/data/archive/2022/02/PXD028735/LFQ_timsTOFPro_PASEF_Condition_B_Sample_Alpha_03.d.zip

# potential reviewers

_No response_

# Will you be able to work on the implementation (coding) yourself, with additional help from the ProteoBench maintainers?

- [X] yes
- [ ] no

#  any other information

_No response_

**Comment by mlocardpaulet** [comment]
I have checked, and we still have some of Bart's 3-species mix in the -20˚C. The same ones that were used for the DDA quant. module. 
We could inject them on our timsTOF SCP. @brvpuyve do you think that this is a good idea? Would this be better? Or are these too old?

**Reply by brvpuyve** [reply]
Hmm tricky, but since these are pretty highly concentrated samples (1 µg/µL) it might be that they are still relatively okay. For the SCP you probably also don't need that much input material (10-20ng on column). Otherwise try to run the QC sample to see how many proteins are still identified?

**Reply by mlocardpaulet** [reply]
How many would you expect (identified proteins in the QC sample)? 
So you think that if we have enough identified proteins in the QC we should go for it? 
Karima (who offered to run the samples here) says that indeed, we don't need much material.

**Reply by brvpuyve** [reply]
The QC is a mixture of sample A and B so I would expect at least 5000 identified proteins and probably even more

**Reply by mlocardpaulet** [reply]
OK, Karima nicely offered to run everything with diaPASEF (including the QC sample) when she has time on the instrument. Maybe next week. 
We can then check the quality of the QC, and if it is good, we can use the raw files for this new module. 
Anything else we need to think of?

**Reply by brvpuyve** [reply]
Perfect! Would be a convenient solution to get some more high quality diaPASEF data.
